app/scripts/views/settings/avatar_camera.js

```diff
@@ -168,8 +181,7 @@ define(function (require, exports, module) {
           return account.uploadAvatar(data);
         })
         .then(function (result) {
-          self.stream.stop();
-          delete self.stream;
+          self.stopAndDestroyStream();
```

Comment:
We can update the test for `submit` to assert that this was called instead of checking for stream, to keep it shallow.